GeekHijabi/More-recipes
# 151354032-favorite-recipe-implementation

What does this PR do?
- User can make a recipe their favorite by clicking on the favorite button
- User can access their favorite recipes by viewing it on a page
Description of Task to be completed?
- Implement page view for user favorite recipe

How should this be manually tested?
- Click on the favorite button and navigate to the favorite page to see which recipe you have favorited as a user
